A required class was missing while executing io.takari.polyglot:polyglot-translate-plugin:0.1.16:translate

Dear all, 

while executing 
mvn io.takari.polyglot:polyglot-translate-plugin:translate -Dinput=/XXX/pom.xml -Doutput=pomt.rb

I am seeing this error:

[ERROR] Failed to execute goal io.takari.polyglot:polyglot-translate-plugin:0.1.16:translate (default-cli) on project standalone-pom: Execution default-cli of goal io.takari.polyglot:polyglot-translate-plugin:0.1.16:translate failed: A required class was missing while executing io.takari.polyglot:polyglot-translate-plugin:0.1.16:translate: org/codehaus/plexus/util/xml/XmlStreamReader

For some reason, mvn downloads

[INFO] --- polyglot-translate-plugin:0.1.16:translate (default-cli) @ standalone-pom ---
Downloading: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/1.1/plexus-utils-1.1.jar
Downloaded: https://repo.maven.apache.org/maven2/org/codehaus/plexus/plexus-utils/1.1/plexus-utils-1.1.jar (165 KB at 244.2 KB/sec)

And that version of plexus-utils does not seem to have the XmlStreamReader Class in its ja.r

Any ideas how to fix this?

Best Regards,

Andreas

I have the same problem with 0.2.0.  Tried the plugin for the first time, and it fails with the above error:

> [ERROR] Failed to execute goal io.takari.polyglot:polyglot-translate-plugin:0.2.0:translate (default-cli) on project core: Execution default-cli of goal io.takari.polyglot:polyglot-translate-plugin:0.2.0:translate failed: A required class was missing while executing io.takari.polyglot:polyglot-translate-plugin:0.2.0:translate: org/codehaus/plexus/util/xml/XmlStreamReader

I tried updating to maven 3.5, but that did not help.

Should this be re-opened?  or create a new issue?

--- a/polyglot-translate-plugin/pom.xml
+++ b/polyglot-translate-plugin/pom.xml
@@ -18,6 +18,13 @@
   <packaging>takari-maven-plugin</packaging>
 
   <dependencies>
+    <!-- This dep was set to provided in parent pom, but when used as Maven plugin, it is essential -->
+    <dependency>
+      <groupId>org.codehaus.plexus</groupId>
+      <artifactId>plexus-utils</artifactId>
+      <version>3.0.24</version>
+      <scope>runtime</scope>
+    </dependency>
     <dependency>
       <groupId>io.takari.polyglot</groupId>
       <artifactId>polyglot-common</artifactId>

